.github/workflows/jit.yml

```diff
@@ -81,7 +81,7 @@ jobs:
             runner: windows-11-arm
           - target: x86_64-apple-darwin/clang
             architecture: x86_64
-            runner: macos-13
+            runner: macos-15-intel
           - target: aarch64-apple-darwin/clang
             architecture: aarch64
             runner: macos-14
```

Comment:
Maybe update arm runners to 15 in a separate follow-up PR (since this one is declared to be about handling intel)